k8s/docker-compose.prod.yaml

```diff
@@ -0,0 +1,33 @@
+---
+version: '3'
+
+volumes:
+    mongodata:
+
+services:
+    database:
+        image: mongo:4.4
+        container_name: mongo_prod
+        environment: 
+            - MONGO_INITDB_DATABASE=master # database name
+            - MONGO_INITDB_ROOT_USERNAME=admin # root username
+            - MONGO_INITDB_ROOT_PASSWORD=secret # root password
```

Comment:
do these get overridden in prod?

Reply:
No, but the DB also isn't exposed to the internet or even to the underlying OS....still maybe I should change this.

Reply:
The OS sees everything... the OS always knows

Reply:
Well...it doesn't know at a level the user can theoretically access if they aren't like ... root. And I feel like if they are root, there are bigger issues.